src/pages/images/ImageSelector.tsx

```diff
@@ -218,6 +257,12 @@ const ImageSelector: FC<Props> = ({ onSelect, onClose }) => {
             "aria-label": "Alias",
             onClick: selectImage,
           },
+          {
+            content: getSource(),
+            role: "rowheader",
```

Comment:
This, and all the others for the other columns, ideally should be:
```suggestion
            role: "cell",
```

Reply:
right, the other cells are only with a wrong role. fixed all of them.